manifest_staging/helm/gatekeeper/README.md

```diff
@@ -0,0 +1,30 @@
+
+## Parameters
+
+| Parameter                 | Description                                                 | Default                                                                    |
+|:--------------------------|:------------------------------------------------------------|:---------------------------------------------------------------------------|
+| auditInterval             | The frequency with which audit is run                       | `60`                                                                       |
+| constraintViolationsLimit | The maximum # of audit violations reported on a constraint  | `20`                                                                       |
+| auditFromCache            | Take the roster of resources to audit from the OPA cache    | `false`                                                                    |
+| logLevel                  | Minimum log level                                           | `INFO`                                                                     |
+| image.pullPolicy          | The image pull policy                                       | `IfNotPresent`                                                             |
+| image.repository          | The Docker image repository                                 | `quay.io/open-policy-agent/gatekeeper`                                     |
+| image.release             | The image release tag to use                                | `v3.1.0-beta.8`                                                            |
+| resources                 | The resource request/limits for the container image         | limits: 1 CPU, 512Mi, requests: 100mCPU, 256Mi                             |
+| nodeSelector              | The node selector to use for pod scheduling                 | `{}`                                                                       |
+| affinity                  | The node affinity to use for pod scheduling                 |                                                                            |
+| tolerations               | The tolerations to use for pod scheduling                   | `[]`                                                                       |
+| replicas                  | The number of Gatekeeper replicas to deploy for the webhook | `1`                                                                        |
+| podAnnotations            | The annotations to add to the Gatekeeper pods               | `container.seccomp.security.alpha.kubernetes.io/manager: runtime/default`  |
+
+
+## Contributing Changes
+
+This Helm chart is autogenerated from the Gatekeeper static manifest. The
+generator code lives under `cmd/build/helmify`. To make modifications to this
+template, please edit `kustomization.yaml` and `replacements.go` under that
+directory and then run `make manifests`. Your changes will show up in the
+`manifest_staging` directory and will be promoted to the root `chart` directory
```

Comment:
it's `chart` here.

Reply:
As opposed to `helm`? Or am I misinterpreting this comment?

If the above is correct, I'll adjust this to whatever the standard is.

Reply:
Updated to `charts`